### ISSUE TITLE
Web accessibility development

### ISSUE THREAD

**Issue by polominsk** [30-seconds/30-seconds-of-code#1127]
I would like to create a session to just talk about web accessibility development.

**Comment by Chalarangelo** [comment]
Hey there. Accessibility has been on our radar as a topic for quite some time, however we currently lack the infrastructure/repos to tackle it. We will eventually open a repository for the topic and/or add some blogs, so stay tuned for that. If you want to contribute, you can follow the org in general and you'll see when/if we start such as project. Thank you.
